service/service/settings/local.py.template

```diff
@@ -0,0 +1,22 @@
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'some_secret'
+
+# Database
+# https://docs.djangoproject.com/en/1.9/ref/settings/#databases
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
```

Comment:
Popped that in tere, but I think now that the sqlite config should go in base.py